lib/mongo/database.rb

```diff
@@ -155,11 +158,11 @@ def collections
     def command(operation, opts = {})
       preference = ServerSelector.get(opts[:read] || ServerSelector::PRIMARY)
       server = preference.select_server(cluster)
-      Operation::Commands::Command.new({
+      Operation::Commands::Command.new(
```

Comment:
It's not just CRUD.  Drivers are supposed to look for operationTime in *all* database responses, including errors.  If that's not updated in the spec, it's supposed to be.